src/main/resources/schema/rpa-json-schema.json

```diff
@@ -705,7 +705,7 @@
           },
           "litigiousPartyID": {
             "$id": "#/definitions/miscellaneousEventType/items/properties/litigiousPartyID",
-            "type": "null"
+            "type": ["null", "string"]
```

Comment:
Just had a discussion with Sankavi, this might not be an accepted solution because it means changing the schema. I'll pop a question to RPA team and come back to this later.